### PR TITLE
doc/man/ceph: document 'ceph config'

### PR DESCRIPTION
**Pull request by tchaikov**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

File: doc/man/8/ceph.rst [hunk 1]
```diff
@@ -174,6 +176,80 @@ Usage::
...
+
+	ceph config ls
+
+Subcommand ``help`` to describe the specified configuration option
+
+Usage::
+
+    ceph config help <option>
+
+Subcommand ``get`` to dump the option(s) for the specific entity.
```
Comment (mchangir) [inline doc/man/8/ceph.rst]: Do we need to define what a *specific entity* is ?

File: doc/man/8/ceph.rst [hunk 2]
```diff
@@ -174,6 +176,80 @@ Usage::
...
+
+Usage::
+
+    ceph config get <who> {<option>}
```
Comment (mchangir) [inline doc/man/8/ceph.rst]: Do we need to define who `<who>` is ?

File: doc/man/8/ceph.rst [hunk 3]
```diff
@@ -174,10 +176,87 @@ Usage::
...
+Usage::
+
+    ceph config rm <who> <option>
+
+Subcommand ``log`` to show recent history of config changes. If `count` option
+is omitted it defeaults to 10.
+
+Usage::
+
+    ceph log {<count>}
```
Comment (sebastian-philipp) [inline doc/man/8/ceph.rst]:
```suggestion
    ceph config log {<count>}
```
? see [config_log](https://ceph-command-api.readthedocs.io/en/latest/mon_command_api.html#ceph_command_api.MonCommandApi.config_log)

File: doc/man/8/ceph.rst [hunk 4]
```diff
@@ -174,10 +176,87 @@ Usage::
...
+
+Subcommand ``reset`` to revert configuration to the specified historical version
+
+Usage::
+
+    ceph reset <version>
```
Comment (sebastian-philipp) [inline doc/man/8/ceph.rst]:
```suggestion
    ceph config reset <version>
```
? See [config_reset](https://ceph-command-api.readthedocs.io/en/latest/mon_command_api.html#ceph_command_api.MonCommandApi.config_reset)

File: doc/man/8/ceph.rst (hunk 3 above)
Comment (tchaikov) [reply doc/man/8/ceph.rst]: fixed.

File: doc/man/8/ceph.rst (hunk 4 above)
Comment (tchaikov) [reply doc/man/8/ceph.rst]: and fixed.

File: doc/man/8/ceph.rst [hunk 5]
```diff
@@ -174,10 +176,87 @@ Usage::
...
+
+    ceph config help <option>
+
+Subcommand ``get`` to dump the option(s) for the specified entity.
+
+Usage::
+
+    ceph config get <who> {<option>}
+
+Subcommand ``set`` to set an option for one or more specified entities
```
Comment (liewegas) [inline doc/man/8/ceph.rst]: Missing the `show` subcommand, which shows the runtime values that a daemon is currently reporting.

File: doc/man/8/ceph.rst [hunk 6]
```diff
@@ -13,6 +13,8 @@ Synopsis
 
 | **ceph** **compact**
 
+| **ceph** **config** [ *dump* | *ls* | *help* | *get* | *set* | *rm* | *log* | *reset* | *assimilate-conf* | *generate-minimal-conf* ] ...
```
Comment (liewegas) [inline doc/man/8/ceph.rst]: missing `show`

File: doc/man/8/ceph.rst (hunk 5 above)
Comment (tchaikov) [reply doc/man/8/ceph.rst]: added `ceph config show` and `ceph config show-with-defaults`.

**Pull request merged**